neuron_reduce/subtree_reductor_func.py

```diff
@@ -707,15 +707,23 @@ def merge_and_add_synapses(num_of_subtrees,
 
     return new_synapses_list, subtree_ind_to_q
 
-
+def textify_seg_to_seg(original_seg_to_reduced_seg):
```

Comment:
I think I'd write this like:
```
def textify_seg_to_seg(segs):
    '''convert segment dictionary to text'''
    ret = {str(k): str(v) for k, v in segs.items()}
    return ret
```

Reply:
So maybe there is no need for a function? and just do it in the subtree_reductor function?

Reply:
I'm not usually a fan of single line functions, but in this case it's useful since it means there doesn't have to be a comment explaining *why* the segments are being textified.

This is a style thing, and that's just my opinion.